solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -369,6 +369,18 @@ contract RandomBeacon is IRandomBeacon, IApplication, Ownable, Reimbursable {
         staking = _staking;
         reimbursementPool = _reimbursementPool;
 
+        require(
+            address(_sortitionPool) != address(0),
```

Comment:
How about "Zero-address sortition pool", "Zero-address T Token" etc? If a revert happens, then the error message is more descriptive of where exactly it happened.

Same comment in all the places in this PR where we have "Zero-address ..."

Reply:
The same message is used to save on the contract size. If we use another messages, we will end up with much more bytecode.